### ISSUE TITLE
Wiki currently doesn't exist

### ISSUE THREAD

**Issue by toothbrush7777777** [mozilla/seasponge#76]
There are several links (now dead) on the homepage to the wiki, which doesn't exist. Can the pages be recovered or created, please?

I'm finding it rather hard to follow the UI, and it would be really helpful if there was some documentation on it.

Thanks

**Comment by Frozenfire92** [comment]
It appears someone has deleted that page :( I wasn't able to find a restore or history option either)

This issue holds the gifs that were available. Will have to look into properly documenting this

**Comment by Glavin001** [comment]
Looks like @kallada deleted some pages: https://github.com/mozilla/seasponge/wiki/_history

**Comment by kallada** [comment]
Yeah - sorry - I was moving the docs to a better documentation service. I'll fix that shortly.

**Comment by kallada** [comment]
@toothbrush7777777 

As far as I can tell there was only one page in the Wiki; It's been restored for now, [check it out](https://github.com/mozilla/seasponge/wiki/Usage).

Don't hesitate to let us know if you have any other issues.

**Comment by Glavin001** [comment]
Thanks for clearing that up, @kallada. Moving the docs may end up helping keep the docs in sync with the code. Once the docs/wiki have migrated into the app repo itself then we can then remove from the wiki. Thanks!

**Comment by toothbrush7777777** [comment]
Thank you very much. Now I'm beginning to understand the flow :relieved:

**Comment by Glavin001** [comment]
@toothbrush7777777 you may also find the video helpful: https://air.mozilla.org/mozilla-winter-of-security-seasponge-a-tool-for-easy-threat-modeling/
